src/visitor/import-export.js

```diff
@@ -123,14 +126,25 @@ class ImportExportVisitor extends Visitor {
     const { declaration } = node
     const { id, type } = declaration
 
-    if (id && (type === "FunctionDeclaration" ||
-               type === "ClassDeclaration")) {
+    if (type === "FunctionDeclaration" || (id && type === "ClassDeclaration")) {
```

Comment:
Here I left the `id` check for classes because, even if tecnically they are declarations, they aren't hoisted and thus the difference between class expressions and declarations isn't noticeable.